.github/workflows/format.yml

```diff
@@ -14,23 +14,22 @@ jobs:
     name: Format (Python)
     runs-on: ubuntu-22.04
     steps:
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.11'
 
-      - name: Install dependencies
-        run:
-            python -m pip install click==8.0.4 black==23.7.0 isort==5.13.2
+      - uses: actions/checkout@v3
 
-      - name: Checkout PennyLane-Lightning
-        uses: actions/checkout@v3
-
-      - name: Run isort
-        run: python -m isort --profile black ./pennylane_lightning/ ./mpitests ./tests --check --diff
+      - name: python-isort
+        uses: isort/isort-action@v1.1.0
```

Comment:
If the action is provided by a trustworthy party, then *it should be* safe for use. I did look through the source code of the action and it seems simple enough, and does what we were doing previously with cli call